strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/AccessModel.java

```diff
@@ -21,7 +21,6 @@
  * @author Alex Oreshkevich
  */
 public class AccessModel
-        extends GenericEntity
```

Comment:
I'm curious why is this changed ? `AccessModel` is a persistent field of entity `org.carlspring.strongbox.users.domain.User`

Reply:
`AccessModel` is `@Embedded` in `org.carlspring.strongbox.users.domain.User`